source/frontend/src/features/mapSideBar/property/MotiInventoryHeader.tsx

```diff
@@ -43,6 +44,16 @@ export const MotiInventoryHeader: React.FunctionComponent<IMotiInventoryHeaderPr
     return false;
   }, [apiProperty]);
 
+  const isDisposed = React.useMemo(() => {
+    if (
+      exists(geoserverMapData?.features[0]) &&
+      geoserverMapData?.features[0]?.properties?.IS_DISPOSED
+    ) {
+      return true;
+    }
+    return false;
+  }, [geoserverMapData?.features]);
+
```

Comment:
**nit:** This code assumes that the features array always has elements - perhaps check for null/empty array to avoid NPE?